Bottom: cc627a383bca7302d2c50b2a8884ebed0e3d1b99
Top:    40319d696db779fa36820523541a712f143c25f4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 16:29:15 +0000

Refresh of refactor-to-use-step_id-not

---

diff --git a/src/hid/common/step_id.h b/src/hid/common/step_id.h
new file mode 100644
index 0000000..86a4974
--- /dev/null
+++ b/src/hid/common/step_id.h
@@ -0,0 +1,2 @@
+typedef int step_id;
+typedef GList* step_id_list;
